### PR TITLE
chore: update setup documentation

### PR DESCRIPTION
**Pull request by ColbyJohnIBM**
#### What do these changes do/fix?

- Add additional prerequisites and instructions
- Add screenshots to help users find the values necessary for config files

#### How do you test/verify these changes?
No testing, just documentation

#### Have you documented your changes (if necessary)?
Yes. 😄 

#### Are there any breaking changes included in this pull request?
None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ First, you will need to customize and improve your document retrieval project on
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [nvm](https://github.com/nvm-sh/nvm#installation-and-update)
+  - Set Node version to 16.x.x (e.g. `16.14.2`) via `nvm install 16.14.2` and `nvm use 16.14.2`
```
Comment (jhpedemonte) [inline README.md:51]: Nit: I think you can just do `nvm install 16` (same for `use`) and it will just use the latest.

File: README.md [hunk 2]
```diff
@@ -48,7 +48,9 @@ First, you will need to customize and improve your document retrieval project on
...
 - [yarn](https://yarnpkg.com/getting-started/install) or [npm](https://www.npmjs.com/get-npm)
+- For MacOS, install any waiting OS Software Updates and run `xcode-select --install` in terminal
```
Comment (jhpedemonte) [inline README.md:53]:
```suggestion
- On MacOS, the Xcode Command Line Tools. To install, run `xcode-select --install` in terminal.
```

Why do we need to specify running updates? I think just `Xcode-select --install` should be fine (even if user is down level on the OS).

File: README.md [hunk 3]
```diff
@@ -145,21 +150,31 @@ yarn workspace discovery-search-app run start
...
 
    ```
    yarn workspace @ibm-watson/discovery-react-components run build
    ```
 
-7. Start the example app:
+7. Start the example server:
+
+   ```
+   yarn workspace discovery-search-app server
```
Comment (jhpedemonte) [inline README.md:165]: Is this correct?
```suggestion
   yarn workspace discovery-search-app run server
```

**Review by jhpedemonte — APPROVED** [review]
Some small suggestions. With those fixed, approved.

File: README.md (hunk 2 above)
Comment (ColbyJohnIBM) [reply README.md:53]: When trying to get it working recently, `xcode-select --install` points to running `Software update` first if there are updates available.

File: README.md (hunk 3 above)
Comment (ColbyJohnIBM) [reply README.md:165]: `yarn server` or `yarn run server` seem to run the same, but I can update to `yarn run server`.

**Pull request merged**